prow/git/v2/client_factory.go

```diff
@@ -264,7 +264,7 @@ func (c *clientFactory) bootstrapClients(org, repo, dir string) (cacher, cloner,
 		dir = workdir
 	}
 	logger := c.logger.WithFields(logrus.Fields{"org": org, "repo": repo})
-	logger.WithField("dir", dir).Debug("Creating a pre-initialized client.")
+	logger.WithField("dir", dir).Info("Creating a pre-initialized client.")
```

Comment:
Was this just for debugging or did you mean to include this? These seem like they might be a bit spammy for the info level, but it doesn't matter much.

Reply:
Oh yeah these were for my own debugging. Will clean it up.